src/components/molecules/copyButton.tsx

```diff
@@ -13,7 +13,7 @@ import { CopyIcon } from "@assets/image/icons";
 
 export const CopyButton = ({
 	className,
-	size = "md",
+	size = "xs",
```

Comment:
You don't need this prop at all, we removed it everywhere.

Reply:
Never mind, your solution is great 🚀 
![Alt Text](https://media4.giphy.com/media/v1.Y2lkPTc5MGI3NjExNGpweTJqOXFkZ2RrcTY0d2hwcG1uNDM5a29yd2JkdGE2bWJwZjE2dCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/JZJfNZ007yOBNeC7J4/giphy.gif)